utm_source_multi_company/security/utm_source.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
```

Comment:
```suggestion
<odoo noupdate="1">
```

Reply:
ignore that last comment. see @pedrobaeza remarks here : https://github.com/OCA/multi-company/pull/506#discussion_r1429125380